src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java

```diff
@@ -75,7 +73,9 @@ public void testLoadFromUrl() throws Exception {
             }
         }
 
-        assumeTrue(actualFilterSet != null, "No Internet connection.");
```

Comment:
assumeXXX should not be replaced with an assert.
This test requires an internet connection, which may not be available.
This can be replaced with `if`, but an `assume` method is better, since it will output the reason to the tests log.

Reply:
I am not understanding why it can't be replaced. Can you provide clarification on why we can't use assume here?

Reply:
false conditions in `assume` methods will not break the build. It simply reports that the test is ignored due to "Text".
false condition in `assert` methods will break the build.